dataeng/resources/snowflake-schema-builder.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -ex
+
+# Setup
+cd $WORKSPACE/warehouse-transforms
+pip install -r tools/dbt_schema_builder/requirements.txt
+
+# Create a new git branch
+now=$(date +%d_%m_%Y_%H_%M_%S)
+branchname="builder_$now"
+git checkout -b "$branchname"
+
+# Run the dbt script to update schemas and sql
+python tools/dbt_schema_builder/schema_builder.py build --schemas $SCHEMAS --project $DBT_PROJECT --profile $DBT_PROFILE --target $DBT_TARGET --profiles-dir $WORKSPACE/analytics-secure/analytics-tasks/
+
+# Commit all changes to the new branch
+git commit -a --message "Schema Builder automated dbt update at $now"
+
+# Create a PR on Github from the new branch
+hub pull-request --push --no-edit
```

Comment:
Maybe a `-r <REVIEWERS>` should be added here as well for notification purposes?